development/webpack/webpack.config.ts

```diff
@@ -178,7 +179,46 @@ const plugins: WebpackPluginInstance[] = [
     ],
   }),
 ];
+
+const optionalRules: RuleSetRule[] = [];
+
 if (args.lavamoat) {
+  const { lavamoatPlugin } = require('./utils/plugins/LavamoatPlugin');
+  const { exclude: excludeLoader } = require('@lavamoat/webpack');
+  optionalRules.push({
+    // 1) the entry module itself
+    test: /bootstrap/u,
+    use: [excludeLoader],
+  });
+  optionalRules.push({
+    // 2) everything issued by that entry (has issuerLayer='bootstrap')
+    issuerLayer: 'bootstrap',
+    use: [excludeLoader],
+  });
+  plugins.push({
+    apply(compiler: Compiler) {
+      compiler.hooks.thisCompilation.tap('Layer', (compilation) => {
+        compilation.hooks.addEntry.tap(
+          'Layer',
+          (entry: Dependency, options) => {
+            const name = options.name;
+            if (
+              name &&
+              'request' in entry &&
+              typeof entry.request === 'string'
+            ) {
+              if (name === 'bootstrap') {
+                const entryData = compilation.entries.get(name);
+                if (entryData) {
+                  entryData.options.layer = 'bootstrap';
+                }
+              }
+            }
+          },
+        );
+      });
+    },
+  });
```

Comment:
This really should be moved to our plugins directory, and then be made more generic (so we could apply it to other files by name/path with one invocation).